extensions/src/org/pentaho/platform/web/http/filters/PentahoWebContextFilter.java

```diff
@@ -369,6 +368,20 @@ private void printResourcesForContext( String contextName, OutputStream out, Htt
 
   }
 
+  public void printRequireJsCfgStart( OutputStream out ) throws IOException {
+
+    int waitTime;
+
+    try {
```

Comment:
Since this is invoked a lot, I'd rather you read the setting from the pentaho.xml and store it in the cache (see CacheManager) and use it from there. If a customer needs to refresh the value, they can clear the cache.